### DISCUSSION TITLE
我想让系统默认英文应该才哪里设置啊？

### DISCUSSION THREAD

**Discussion by zz99zz9** [ant-design/ant-design#41528] [General]
目前后台系统默认中文，需要切换一下才是英文，如果初始化成英文呢？

**Comment by Yuiai01** [comment]
可以通过 `ConfigProvider` ,可以看看这个 https://ant.design/components/config-provider-cn#%E4%BB%A3%E7%A0%81%E6%BC%94%E7%A4%BA

**Reply by zz99zz9** [reply]
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/22146191/228482518-0aaba9fc-7849-47fb-9f61-4186287f6252.png">
整体已经默认英文了，可是表单却还是默认中文
